traveler-ios-ui/TravelerKitUI/ViewControllers/WishlistViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  WishlistViewController.swift
+//  TravelerKitUI
+//
+//  Created by Ben Ruan on 2019-09-04.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+open class WishlistViewController: UIViewController {
+    public var query: WishlistQuery?
+    private var error: Error?
+    private var result: WishlistResult?
+
+    override open func viewDidLoad() {
+        super.viewDidLoad()
+
+        reload()
+    }
+
+    private func reload() {
+        guard let query = query else {
+            Log("No WishlistQuery", data: nil, level: .error)
+            return
+        }
+
+        Traveler.fetchWishlist(query, identifier: nil, delegate: self)
+        DispatchQueue.main.async {
```

Comment:
What's the need of using DispatchQueue here?. Also, you want to perform the loading segue BEFORE making the call to traveler.

Reply:
It's interesting that if I make the call to traveler before performing the loading segue, I have to perform with the dispatch queue.

Reply:
This is unaddressed